plugins/package-managers/nuget/src/funTest/assets/nuget-expected-output.yml

```diff
@@ -27,7 +27,34 @@ project:
       - timestamp: "1970-01-01T00:00:00Z"
         source: "NuGet"
         message: "Failed to get remote metadata for name: 'foobar' version: '1.2.3'.\
-          \ System.Exception: No package metadata found for foobar.1.2.3.\n   at NugetInspector.NugetApi.FindPackageVersion(PackageIdentity\
+          \ System.Exception: No package metadata found for foobar.1.2.3.\n    System.AggregateException:\
```

Comment:
Do you have any idea what could have caused this change in error message?

Reply:
Please see the commit message: "It is unclear what caused the message to change..."